### PR TITLE
Edit UG 

### LINKED ISSUE #105: UG Advise by Ms Anita
- Change heading "Who is our audience?" to "Who is this app for?"
- Move the heading above to DG

### PR DESCRIPTION
**Pull request by harithadiv**
- Edited calendar related classes names to make it more appropriate 
- Edited target audience, FAQ and command summary in UG
- Edited calendar to be after event in UG (flow)
- Make most strings to private final static 
- Update help function to include calendar 
- Adjusted flow of help function to match UG

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -70,23 +71,22 @@ As long as you need to keep track of upcoming events or class, you are our audie
...
-    <INSERT SCREENSHOT>
-1.  You should see this screen if everything is successful.
 
 ### 2.2 Startup using Command Line
 1.  Open your terminal.
 1.  Navigate to the home folder containing Pac.
 1.  cd followed by the file path into the terminal as shown below
     <INSERT IMAGE> 
 1.  Type `java -jar` followed by the name of the jar file and press **Enter**.
+1.  You should see this screen if everything is successful.
```
Comment (lowjiayee) [inline docs/UserGuide.md]: Is there any picture added after this line?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -70,23 +71,22 @@ As long as you need to keep track of upcoming events or class, you are our audie
...
 
 ### 2.3 Startup using JAR
 1.  Open home folder containing Pac.
 1.  Run the packaged JAR file by double clicking it 
     and a window should appear in a few seconds.
     <INSERT IMAGE>
-1. Once opened, type in the command when prompted and press **Enter**.
+1.  Once opened, type in the command when prompted and press **Enter**.
+1.  You should see this screen if everything is successful.
```
Comment (lowjiayee) [inline docs/UserGuide.md]: Is there any picture added after this line?

File: docs/UserGuide.md (hunk 1 above)
Comment (harithadiv) [reply docs/UserGuide.md]: yes. I'll add it now

**Review by benchan911 — COMMENTED** [review]
For the images that we include from the command line interface, we should make sure that the images are consistent in their background before submission so that it looks more like a team's work instead of many individuals. Just writing here first.

**Review by benchan911 — APPROVED** [review]
Yes, please merge :)

**Pull request merged**